### PR TITLE
Add information about multi-arch image to README

### PR DESCRIPTION
**Pull request by barthy1**
Also update to the correct one information about required env variables
for multi-arch build

Signed-off-by: Yulia Gaponenko <yulia.gaponenko1@de.ibm.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@TomSweeneyRedHat @vrothberg PTAL

File: contrib/skopeoimage/README.md [hunk 1]
```diff
@@ -16,6 +16,13 @@ using the latest Fedora and then Skopeo is installed into them:
   * quay.io/skopeo/upstream - This image is built using the latest code found in this GitHub repository.  When someone creates a commit and pushes it, the image is created.  Due to that the image changes frequently and is not guaranteed to be stable.  Built with skopeoimage/upstream/Dockerfile.
   * quay.io/skopeo/testing - This image is built using the latest version of Skopeo that is or was in updates testing for Fedora.  At times this may be the same as the stable image.  This container image will primarily be used by the development teams for verification testing when a new package is created.  Built with skopeoimage/testing/Dockerfile.
 
+## Multiarch images
+
+Multiarch images are available for skopeo upstream and stable versions. Supported architectures are `amd64`, `s390x`, `ppc64le`.
```
Comment (TomSweeneyRedHat) [inline contrib/skopeoimage/README.md:21]: nit
```suggestion
Multiarch images are available for Skopeo upstream and stable versions. Supported architectures are `amd64`, `s390x`, `ppc64le`.
```

File: contrib/skopeoimage/README.md [hunk 2]
```diff
@@ -16,6 +16,13 @@ using the latest Fedora and then Skopeo is installed into them:
...
+Available images are `quay.io/skopeo/upstream:master`, `quay.io/skopeo/stable:v1.2.0`, `quay.io/containers/skopeo:v1.2.0`.
+
+Images can be used the same way as in single architecture case, no extra setup is required. For samples see next chapter.
```
Comment (TomSweeneyRedHat) [inline contrib/skopeoimage/README.md:24]:
```suggestion
Images can be used the same way as in a single architecture case, no extra setup is required. For samples see next chapter.
```

File: release/README.md [hunk 3]
```diff
@@ -22,12 +22,13 @@ Several environment variables are used to customize image names and keep private
 
 **Image tags** are specified in environment variable and should be manually updated in case of new release.
 
-- `QUAY_USERNAME` and `QUAY_PASSWORD` are credentials to push image to corresponding quay.io repositories, should have write permissions. These variables should be specified in [Travis](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings).
+- `SKOPEO_QUAY_USERNAME` and `SKOPEO_QUAY_PASSWORD` are credentials to push images to `quay.io/skopeo/stable` and `quay.io/skopeo/upstream` repos, should have write permissions. These variables should be specified in [Travis](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings).
```
Comment (TomSweeneyRedHat) [inline release/README.md:25]:
```suggestion
- `SKOPEO_QUAY_USERNAME` and `SKOPEO_QUAY_PASSWORD` are credentials to push images to `quay.io/skopeo/stable` and `quay.io/skopeo/upstream` repos, and require the credentials to have write permissions. These variables should be specified in [Travis](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings).
```

File: release/README.md [hunk 4]
```diff
@@ -22,12 +22,13 @@ Several environment variables are used to customize image names and keep private
...
+- `CONTAINERS_QUAY_USERNAME` and `CONTAINERS_QUAY_PASSWORD` are credentials to push image to `quay.io/containers/skopeo` repo, should have write permissions. These variables should be specified in [Travis](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings).
```
Comment (TomSweeneyRedHat) [inline release/README.md:26]:
```suggestion
- `CONTAINERS_QUAY_USERNAME` and `CONTAINERS_QUAY_PASSWORD` are credentials to push images to `quay.io/containers/skopeo` repos, and require the credentials to have write permissions. These variables should be specified in [Travis](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings).
```

File: release/README.md [hunk 5]
```diff
@@ -22,12 +22,13 @@ Several environment variables are used to customize image names and keep private
...
 
 Variables in .travis.yml
 - `MULTIARCH_MANIFEST_ARCHITECTURES` is a list with architecture shortnames, to apprear in final multiarch manifest. The values should fit to architectures used in the `image-build-push` Travis step.
-- `STABLE_IMAGE`, `EXTRA_STABLE_IMAGE` are image names to publish stable skopeo
-- `UPSTREAM_IMAGE` is image name to publish upstream skopeo
+- `STABLE_IMAGE`, `EXTRA_STABLE_IMAGE` are image names to publish stable skopeo.
+- `UPSTREAM_IMAGE` is image name to publish upstream skopeo.
```
Comment (TomSweeneyRedHat) [inline release/README.md:31]:
```suggestion
- `UPSTREAM_IMAGE` is an image name to publish upstream Skopeo.
```

File: release/README.md [hunk 6]
```diff
@@ -22,12 +22,13 @@ Several environment variables are used to customize image names and keep private
...
+- `SKOPEO_QUAY_USERNAME` and `SKOPEO_QUAY_PASSWORD` are credentials to push images to `quay.io/skopeo/stable` and `quay.io/skopeo/upstream` repos, should have write permissions. These variables should be specified in [Travis](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings).
+- `CONTAINERS_QUAY_USERNAME` and `CONTAINERS_QUAY_PASSWORD` are credentials to push image to `quay.io/containers/skopeo` repo, should have write permissions. These variables should be specified in [Travis](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings).
 
 Variables in .travis.yml
 - `MULTIARCH_MANIFEST_ARCHITECTURES` is a list with architecture shortnames, to apprear in final multiarch manifest. The values should fit to architectures used in the `image-build-push` Travis step.
-- `STABLE_IMAGE`, `EXTRA_STABLE_IMAGE` are image names to publish stable skopeo
-- `UPSTREAM_IMAGE` is image name to publish upstream skopeo
+- `STABLE_IMAGE`, `EXTRA_STABLE_IMAGE` are image names to publish stable skopeo.
```
Comment (TomSweeneyRedHat) [inline release/README.md:30]:
```suggestion
- `STABLE_IMAGE`, `EXTRA_STABLE_IMAGE` are image names to publish stable Skopeo.
```

**Comment by TomSweeneyRedHat** [comment]
@barthy1 a few tiny nits scattered about, otherwise it looks great.  Thanks so much for whipping this up so quickly!

**Comment by TomSweeneyRedHat** [comment]
and now needs a rebase to boot.

**Comment by barthy1** [comment]
@TomSweeneyRedHat thank you for review. 
Done :)

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM
Thanks @barthy1 !

**Pull request merged**